src/screens/OtherProfile/index.js

```diff
@@ -164,13 +158,6 @@ const OtherProfile = () => {
   }, []);
 
   React.useEffect(() => {
-    netInfo.addEventListener((state) => {
-      if (state.isConnected) {
-        initData();
-      } else {
-        handleOfflineMode();
-      }
-    });
     initData();
   }, []);
 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the network status check before initializing data might lead to issues when the device is offline. Ensure that the network status check is handled elsewhere or consider reintroducing it.



<!-- This is an auto-generated comment by OSS CodeRabbit -->